test/integration/model/update.test.js

```diff
@@ -1,21 +1,17 @@
 'use strict';
 
 /* jshint -W030 */
-var Support = require(__dirname + '/../support')
-  , DataTypes = require(__dirname + '/../../../lib/data-types')
-  , chai = require('chai')
-  , expect = chai.expect
-  , Support = require(__dirname + '/../support')
-  , current = Support.sequelize
-  , _ = require('lodash');
+const Support = require(__dirname + '/../support');
+const DataTypes = require(__dirname + '/../../../lib/data-types');
+const chai = require('chai');
+const expect = chai.expect;
+const current = Support.sequelize;
+const _ = require('lodash');
 
-describe(Support.getTestDialectTeaser('Model'), function() {
+describe(Support.getTestDialectTeaser('Model'), function () {
   describe('update', function () {
-
-    var Account;
-
     beforeEach(function() {
-      Account = this.sequelize.define('Account', {
+      this.Account = this.sequelize.define('Account', {
```

Comment:
What's the benefit of using the shared test context over a scoped variable? You don't get any help from your editor for the text context